### ISSUE TITLE
/home/yunohost.multimedia/share/Game: open for read: permission denied

### ISSUE THREAD

**Issue by tr1p0p** [YunoHost-Apps/retroarch_ynh#26]
It's seem the Game folder on the yunohost.multimedia not create properly and don't give user permission to add or even open the folder on thing like Nextcloud and all. 

That's making the folder unusable on program like Filezilla and all, this only accessible with sudo on linux terminal (so hard to upload things)

**Comment by Krakinou** [comment]
Hello,
You're right, it seems that all folder into yunohost.multimedia are supposed to be open for the multimedia group, but link created via the helpers are not.

I assume that you may make it available to nextcloud via following command line:
````
chown www-data:multimedia /opt/yunohost/retroarch/assets/cores/Game
````

I cannot patch the script right now as I'm currenlty moving and all my computers, pis and so on are packed into boxes, but I will update it after new year eve when everything will be back online.

Thanks for pointing the issue.

**Comment by Krakinou** [comment]
The retroarch folder did not have the x right for other, this prevented the link to work.
solved by #28
